src/main/java/pl/javastart/task/Triangle.java

```diff
@@ -0,0 +1,15 @@
+package pl.javastart.task;
+
+public class Triangle {
+    private final double base;
+    private final double height;
+
+    public Triangle(double base, double height) {
+        this.base = base;
+        this.height = height;
+    }
+
+    public double calculateTrianglePerimeter() {
+        return 0.5 * this.base * this.height;
```

Comment:
obwód trójkąta to 3*A, albo A+B+C, wysokość bierzemy przy polu

Reply:
Tak oczywiście z rozpędu liczyłem wszędzie pola i nie spojrzałem, że tu liczyliśmy obwód :)